pkg/config/config.go

```diff
@@ -301,6 +316,13 @@ func (c *Config) validate() error {
 		return fmt.Errorf("unsupported namespaceOwnership value %v", c.Ingress.AdmissionPolicy.NamespaceOwnership)
 	}
 
+	if c.Ingress.Ports.Http != nil && (*c.Ingress.Ports.Http < 1 || *c.Ingress.Ports.Http > math.MaxUint16) {
```

Comment:
These conditions should be tested in the unit tests.

Reply:
Done